types/language-tags/Tag.d.ts

```diff
@@ -63,22 +63,22 @@ declare class Tag {
     /**
      * Shortcut for `tag.find('language')`.
      */
-    language(): Subtag;
+    language(): ReturnType<Tag['find']>;
```

Comment:
```suggestion
    language(): Subtag | undefined;
```

Reply:
I thought it might be nice to explicitly express that the `find` derivatives are matching the `find` return type, but I've updated them to `Subtag | undefined`.

I missed that I could just accept your suggestion via the web UI.